instruqt-tracks/multi-cloud-service-networking-with-consul/11-provision-consul-esms/solve-cloud-client

```diff
@@ -11,6 +11,56 @@ vault write consul/roles/esm policies=consul-esm
 
 #tf
 terraform apply -auto-approve 2>&1 | tee terraform.out
-sleep 300
```

Comment:
Stuff sometimes takes longer than 300 seconds to start, so let's actually wait until everything is ready instead of assuming it will be in 300 seconds.